ui/components/app/snaps/update-snap-permission-list/update-snap-permission-list.js

```diff
@@ -29,6 +29,21 @@ export default function UpdateSnapPermissionList({
           );
         },
       )}
+      {getWeightedPermissions(t, revokedPermissions, targetSubjectMetadata).map(
+        (permission, index) => {
+          return (
```

Comment:
This `return` is unnecessary. You can reduce nesting by removing it.
```suggestion
        (permission, index) => (
```

Reply:
Fixed